frameworks/kafka/universe/config.json

```diff
@@ -71,6 +61,26 @@
             "type": "string",
             "enum":  ["OFF", "FATAL", "ERROR", "WARN", "INFO", "DEBUG", "TRACE", "ALL"],
             "default": "INFO"
+          },
+          "security": {
+            "type": "object",
+            "properties": {
+                "tls": {
+                  "description": "Enable TLS support (requires Enterprise DC/OS running in strict security mode).",
+                  "type": "boolean",
+                  "default": false
+                },
+                "tls_allow_plaintext": {
+                  "description": "Keep the plaintext port open when the TLS is enabled. Allows clients to connect using a non-encrypted connection and takes effect only if TLS is enabled.",
+                  "type": "boolean",
+                  "default": false
+                },
+                "tls_enable_authn": {
+                  "description": "Enable TLS authentication (requires TLS to be enabled to take affect).",
```

Comment:
Do we check this requirement at any point?

Reply:
I poked around JSON schema to see if it could do it for us. It doesn't seem to be able to? 

It's also hard to understand for me, a mere mortal.